client/client.go

```diff
@@ -126,15 +138,26 @@ func NewClient(cfg *config.Config) (*Client, error) {
 	// Create a logger
 	logger := log.New(cfg.LogOutput, "", log.LstdFlags)
 
+	resourceUsage, err := stats.NewRingBuff(60)
```

Comment:
Make `60 ` a const with a comment. I thought we were going to make this settable by the user?